### PR TITLE
Update JobTitle.md

### PR DESCRIPTION
**Pull request by gliu6**

### CODE REVIEW

File: JobTitle.md [hunk 1]
```diff
@@ -20,26 +20,28 @@ Parameters (query/form):
 -        title (required if description is empty) : job title
 -        description (required if title is empty) : job description
 -        contentLang : the language of the provided text. You should provide this if it is known, as it will improve the accuracy of our translation system. If this parameter is not specified, the service will attempt to detect the language of the text automatically. Accepted language codes are as follows: ar, bg, ca, zhCHS, zhCHT, cs, da, nl, en, et, tl, fi, fr, de, el, ht, he, hi, hu, id, it, ja, kn, ko, lv, lt, ms, mt, no, fa, pl, pt, ro, ru, sk, sl, es, sv, th, tr, uk, ur, vi, cy.
--        outputType : response format, defaults to json; allowable values are json, xml
+-        outputType (optional): response format, defaults to json; We used to support xml, but by 11/22/2016, xml is abandoned.
```
Comment (msundlof) [inline JobTitle.md]: I would say: "As of 11/22/2016, xml is no longer supported"

File: JobTitle.md [hunk 2]
```diff
@@ -20,26 +20,28 @@ Parameters (query/form):
...
  
-Example: https://api.careerbuilder.com/core/classifier/jobtitle?taxonomy=onet15&title=janitor&contentLang=en&outputType=xml
+Example: https://api.careerbuilder.com/core/classifier/jobtitle?taxonomy=onet17&title=janitor&contentLang=en
```
Comment (thomasmahoney) [inline JobTitle.md]: let's actually just update this to reflect the recent changes and deprecations. fix the casing on contentLang here and above, remove the outputType param altogether, and remove references to the outdated taxonomies (everything before carotene v3.x and onet17)

**Comment by gliu6** [comment]
updated, but I guess I won't merge the pr until we got BSed JTS deployed successfully.

File: JobTitle.md [hunk 3]
```diff
@@ -16,44 +16,74 @@ _________
 
 HTTP method: GET or POST
 Parameters (query/form):
--        taxonomy (required) : classification taxonomy to use; accepted values are onet15, onet17, carotenev1, carotenev2, carotenev2_2, carotenev3, and carotenev3_1. Complete taxonomy lists can be found [here](https://github.com/cbdr/DataScienceAPITaxonomies/tree/master/JobTitle) (restricted to CBReadOnly).
+
+(This API is in transition to match our data science API input/output standard. To provide peaceful transition, we support versions 0.9 and 1.0. Version is passed in the Accept header as the parameter version (Ex. Accept:application/json;version=1.0). 0.9 is the default version if user does not specify explicitly and all our existing users use this version by default. We highly encourage new users to pass version 1.0 in the Accept header. In version 0.9, all input parameters and output json fields are in camel case, while in version 1.0, input parameters and output json fields are in lowercase with underscores.)
```
Comment (thomasmahoney) [inline JobTitle.md]: See my comment from the resolvedtitle PR

File: JobTitle.md [hunk 4]
```diff
@@ -16,44 +16,74 @@ _________
...
--        outputType : response format, defaults to json; allowable values are json, xml
- 
-Example: https://api.careerbuilder.com/core/classifier/jobtitle?taxonomy=onet15&title=janitor&contentLang=en&outputType=xml
+-        contentLang (use contentLang in version 0.9 and use content_lang in version 1.0) (optional) : the language of the provided text. You should provide this if it is known, as it will improve the accuracy of our translation system. If this parameter is not specified, the service will attempt to detect the language of the text automatically. Accepted language codes are as follows: ar, bg, ca, zhCHS, zhCHT, cs, da, nl, en, et, tl, fi, fr, de, el, ht, he, hi, hu, id, it, ja, kn, ko, lv, lt, ms, mt, no, fa, pl, pt, ro, ru, sk, sl, es, sv, th, tr, uk, ur, vi, cy.
+
+Version 1.0 Example (Highly encouraged to use version 1.0): https://api.careerbuilder.com/core/classifier/jobtitle?title=Janitor&taxonomy=carotenev3.1|Onet17&contentLang=auto
```
Comment (thomasmahoney) [inline JobTitle.md]: "Onet17" not sure if these are case-sensitive or not? I always write "onet17" though. and it looks like you left it as "contentLang" even though this is supposed to be a v1.0 example!

File: JobTitle.md [hunk 5]
```diff
@@ -16,44 +16,74 @@ _________
...
+
+(This API is in transition to match our data science API input/output standard. To provide peaceful transition, we support versions 0.9 and 1.0. Version is passed in the Accept header as the parameter version (Ex. Accept:application/json;version=1.0). 0.9 is the default version if user does not specify explicitly and all our existing users use this version by default. We highly encourage new users to pass version 1.0 in the Accept header. In version 0.9, all input parameters and output json fields are in camel case, while in version 1.0, input parameters and output json fields are in lowercase with underscores.)
+-        taxonomy (required) : classification taxonomy to use; accepted values are onet17, carotenev3, and carotenev3_1. Complete taxonomy lists can be found [here](https://github.com/cbdr/DataScienceAPITaxonomies/tree/master/JobTitle) (restricted to CBReadOnly). Note you could supply more than one taxonomy separated by pipe (see example query below).
```
Comment (thomasmahoney) [inline JobTitle.md]: I would rephrase the last sentence - something like "A single title/description may be classified against multiple taxonomies in a single request by providing multiple taxonomies in this parameter, separated by the pipe ("|") character."

File: JobTitle.md [hunk 6]
```diff
@@ -16,44 +16,74 @@ _________
...
+                "minor_title": "Building Cleaning and Pest Control Workers",
+                "minor_id": "2000"
+            }
+        ]
+    }
+}
+```
+
+
+Version 0.9 Example: https://api.careerbuilder.com/core/classifier/jobtitle?taxonomy=onet17&title=janitor&contentLang=en
```
Comment (thomasmahoney) [inline JobTitle.md]: I would just take out the v0.9 example, since it's going away soon anyway. Anybody who's reading the docs must either be writing new code against JTS or modifying some existing code -- either way, they should be looking to upgrade to v1.0 :)

File: JobTitle.md (hunk 4 above)
Comment (gliu6) [reply JobTitle.md]: You are right.

File: JobTitle.md (hunk 6 above)
Comment (gliu6) [reply JobTitle.md]: Yeah, customers always want something easier. And i gotta make 1.0 easier than 0.9.

File: JobTitle.md [hunk 7]
```diff
@@ -17,13 +17,13 @@ _________
...
--        taxonomy (required) : classification taxonomy to use; accepted values are onet17, carotenev3, and carotenev3_1. Complete taxonomy lists can be found [here](https://github.com/cbdr/DataScienceAPITaxonomies/tree/master/JobTitle) (restricted to CBReadOnly). Note you could supply more than one taxonomy separated by pipe (see example query below).
+(This API is in transition to match CareerBuilder API standards. To provide peaceful transition, we support versions 0.9 and 1.0. Version is passed in the Accept header as the parameter version (Ex. Accept:application/json;version=1.0). 0.9 is the default version if user does not specify explicitly and all our existing users use this version by default. We highly encourage new users to pass version 1.0 in the Accept header. In version 0.9, all input parameters and output json fields are in camel case, while in version 1.0, input parameters and output json fields are in lowercase with underscores. Note that 0.9 is planed to be retired in Q1, 2017.)
```
Comment (thomasmahoney) [inline JobTitle.md]: typo in here near the end ("planed"). I would rephrase that sentence as "Version 0.9 is scheduled for retirement by the end of Q1 2017."

**Comment by thomasmahoney** [comment]
:+1:
